### PR TITLE
Pol 248 new resource table 10

### PR DESCRIPTION
**Pull request by KVivek03**
### Description

New Resource Table 10

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Comment by KVivek03** [comment]
Hi @cdwilhelm , the two policies : **cost/rightlink_rightsize/rightlink_rightsize.pt** and **cost/rightlink_rightsize/rightlink-rightsize-add-tags.pt** are in the same folder having only one changelog and ReadMe file.

**Comment by cdwilhelm** [comment]
> Hi @cdwilhelm , the two policies : **cost/rightlink_rightsize/rightlink_rightsize.pt** and **cost/rightlink_rightsize/rightlink-rightsize-add-tags.pt** are in the same folder having only one changelog and ReadMe file.

@KVivek03 yes, that is correct.  The two policies work together and described in the single README

File: cost/google/cloudsql_rightsizing/google_cloudsql_rightsizing.pt [hunk 1]
```diff
@@ -351,21 +363,46 @@ end
...
+      resource_level true
+      field "projectId" do
+        label "Project ID"
+      end
+      field "id" do
+        label "Database ID"
+        path "database_id"
+      end
+      field "region" do
+        label "Region"
```
[the remark this answers is not in the document]
Comment (KVivek03) [reply cost/google/cloudsql_rightsizing/google_cloudsql_rightsizing.pt]: removed the region field, as Zone filed is already present.

File: cost/google/cud_expiration/google_cud_expiration_report.pt [hunk 2]
```diff
@@ -153,20 +153,39 @@ end
...
-###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})
-EOS
     escalate $report_list_of_CUDs
     check eq(size(data),0)
+    export do
+      resource_level true
+      field "projectId" do
+        label "Project ID"
+      end
+      field "id" do
```
[the remark this answers is not in the document]
Comment (KVivek03) [reply cost/google/cud_expiration/google_cud_expiration_report.pt]: moved the id field to the last

File: cost/google/cud_report/google_committed_usediscount_report.pt [hunk 3]
```diff
@@ -127,24 +127,44 @@ EOS
...
-###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})
-EOS
 	escalate $report_list_of_CUDs
 	check eq(size(data),0)
+    export do
+      resource_level true
+      field "projectId" do
+        label "Project ID"
+      end
+      field "id" do
```
[the remark this answers is not in the document]
Comment (KVivek03) [reply cost/google/cud_report/google_committed_usediscount_report.pt]: moved the id field to the last

File: cost/google/cud_report/google_committed_usediscount_report.pt [hunk 4]
```diff
@@ -127,24 +127,44 @@ EOS
...
+        label "Zone"
+      end
+      field "status" do
+        label "Status"
+      end
+      field "plan" do
+        label "Plan"
+      end
+      field "startTimestamp" do
+        label "StartTimestamp"
```
[the remark this answers is not in the document]
Comment (KVivek03) [reply cost/google/cud_report/google_committed_usediscount_report.pt]: changed the label as Start Date

File: cost/google/cud_report/google_committed_usediscount_report.pt [hunk 5]
```diff
@@ -127,24 +127,44 @@ EOS
...
+      end
+      field "endTimestamp" do
+        label "EndTimestamp"
```
[the remark this answers is not in the document]
Comment (KVivek03) [reply cost/google/cud_report/google_committed_usediscount_report.pt]: changed the label as End Date

**Comment by cdwilhelm** [comment]
Looks good, thank you!

**Comment by cdwilhelm** [comment]
@KVivek03 can you please fix the file conflicts on this PR

**Comment by KVivek03** [comment]
Hi @cdwilhelm , I have resolved the merge conflicts.

**Comment by KVivek03** [comment]
hi @cdwilhelm , I am not able to add the "Automatic Actions" parameter details in the README file as i am getting multiple similar headers error in this file-cost/rightlink_rightsize/README.md. Thanks

**Pull request merged**